operations/addresses.md

```diff
@@ -0,0 +1,231 @@
+| `Client` | string | required | Name and version of the client application. |
+| `AccountIds` | array of string | required, max 1000 items | Unique identifiers of [Companies](companies.md#company) or [Customers](customers.md#customer) within the enterprise. |
+| `AddressIds` | array of string | optional, max 1000 items | Unique identifiers of [Addresses](configuration.md#address) within the enterprise. Use this property if you want to fetch specific addresses. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of address items returned. |
+
+### Response
+
+```javascript
+{
+    "Addresses": [
+        {
+            "Id": "fc7b2df3-de66-48a6-907d-af4600ecd892",
+            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "AccountType": "Customer",
+            "Line1": "Rheinlanddamm 207-209",
+            "Line2": null,
+            "City": "Dortmund",
+            "PostalCode": "44137",
+            "CountryCode": "DE",
+            "CountrySubdivisionCode": null,
+            "Latitude": null,
+            "Longitude": null
+        }
+    ],
+    "Cursor": "a5347333-74c0-4ffb-82d7-af4600df5047"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Addresses` | array of [Address items](#address-items) | required | The collection of address items, containing address and account information. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest address item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older address items. |
+
+#### Address items
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the [Address](configuration.md#address). |
+| `AccountId` | string | required | Unique identifier of a [Company](companies.md#company) or a [Customer](customers.md#customer) within the enterprise. |
+| `AccountType` | string | required | A discriminator specifying the [type of account](#account-type), e.g. customer or company. |
+| `Line1` | string | optional | First line of the address. |
+| `Line2` | string | optional | Second line of the address. |
+| `City` | string | optional | The city. |
+| `PostalCode` | string | optional | Postal code. |
+| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
+| `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Latitude` | number | optional | The latitude. |
+| `Longitude` | number | optional | The longitude. |
+
+#### Account type
+
+* [Company](companies.md#company)
+* [Customer](customers.md#customer)
+
+## Add addresses
+
+Adds a new address to the system and assigns it to a specified account. The account has to be within the same enterprise.
```

Comment:
Again, as we can add _multiple_ addresses with one call, would it be better to say "Add one or more new addresses to the system and assign them to specified accounts. Note that an account has to be within the same enterprise as its address."

Reply:
This part should be enough `Add one or more new addresses to the system and assign them to specified accounts.`